README.md

```diff
@@ -28,3 +28,16 @@ To start the development server run:
 ````
 npm run dev
 ````
+
+To seed local DB for dev environment:
+````
+cd seed
+node item-seeder.js
+````
+
+Start mongo shell / view local DB:
```

Comment:
I'd probably specify that this is if you're using MacOS. Though it's pretty obvious if you're a developer that sudo is a mac command, beginners may be confused.

All in all we should probably ignore it for this project since we're time-crunched, but say someone was using Windows OS (as I am), their process to start MongoDB locally would be different (I use `net start MongoDB` and view the databases using MongoDB Compass Community).

Reply:
I'm going to omit it. Threw it there to quickly make you both aware of the DB name to use. If you're running the app, like you noted, it's probably something you don't need to be walked through.

Reply:
It's good to include in a public project, but seeing as this just for our school assignment this month, we can definitely omit it.